jdaviz/configs/default/plugins/markers/tests/test_markers_plugin.py

```diff
@@ -130,9 +132,12 @@ def test_markers_cubeviz(tmp_path, cubeviz_helper, spectrum1d_cube):
 
     # appears as option in export plugin and exports successfully
     assert "Markers: table" in exp.plugin_table.choices
-    exp.filename = str(tmp_path / "cubeviz_export.ecsv")
+    filename = str(tmp_path / "cubeviz_export.ecsv")
+    exp.filename.auto = False
+    exp.filename.value = filename
```

Comment:
This isn't implemented in 3.9 yet, so will need to go back to the original version

```suggestion
    exp.filename = filename
```